lib/elixir_internal_certificate/scrapper/scrappers.ex

```diff
@@ -0,0 +1,32 @@
+defmodule ElixirInternalCertificate.Scrapper.Scrappers do
+  @moduledoc """
+  The Scrappers context.
+  """
+
+  alias ElixirInternalCertificate.Repo
+  alias ElixirInternalCertificate.Scrapper.Schemas.UserSearch
+
+  def insert_search_keywords(attrs),
+    do: Repo.insert_all(UserSearch, attrs)
+
+  def create_search_keyword(keywords, user),
+    do:
```

Comment:
Based on our convention, it should be `do...end` block
![Screen Shot 2565-06-27 at 3 42 31 PM](https://user-images.githubusercontent.com/6076762/175897896-dd765631-0b48-4581-a2e4-41d773ef791a.png)